src/network/bmproto.py

```diff
@@ -269,16 +296,19 @@ def decode_simple(self, char="v"):
             raise BMProtoInsufficientDataError()
 
     def bm_command_error(self):
+        """Decode an error message and log it"""
+        # pylint: disable=unused-variable
         fatalStatus, banTime, inventoryVector, errorText = self.decode_payload_content("vvlsls")
         logger.error("%s:%i error: %i, %s", self.destination.host, self.destination.port, fatalStatus, errorText)
         return True
 
     def bm_command_getdata(self):
+        """Decode object data, conditionally append a newly created object to the write buffer"""
         items = self.decode_payload_content("l32s")
-        # skip?
+        # .. todo:: skip?
         if time.time() < self.skipUntil:
             return True
-        #TODO make this more asynchronous
+        # .. todo:: make this more asynchronous
```

Comment:
This however IS a todo. This loop may take a while, during which time a lot of other stuff is locked. Should be refactored in the future.

Reply:
BTW, the rST syntax so that the sphinx documentation can both format and collect a list of all todo items.